### PR TITLE
Add a note to include timezone for cron jobs

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1658827

### CODE REVIEW

**Comment by mburke5678** [comment]
Maciej Szulik, who implemented the feature in OpenShift says:
"not quite UTC, they are all in server's TZ"
https://coreos.slack.com/archives/CB48XQ4KZ/p1545253590677000
Should we note this in the docs, or would this be assumed?

**Comment by mburke5678** [comment]
@soltysh PTAL

**Review by soltysh — APPROVED** [review]
lgtm

**Comment by xltian** [comment]
@wzheng1 please review

**Comment by wzheng1** [comment]
@xingxingxia Could you please review? Thanks!

File: dev_guide/cron_jobs.adoc [hunk 1]
```diff
@@ -88,6 +88,11 @@ spec:
 3. Sets a label for jobs spawned by this cron job.
 4. The restart policy of the pod. This does not apply to the job controller. See xref:../dev_guide/jobs.adoc#creating-a-job-known-issues[Known Issues and Limitations] for details.
 
+[NOTE]
+====
+All cron job `schedule` times are based the timezone of the master where the job is initiated.
```
Comment (xingxingxia) [inline dev_guide/cron_jobs.adoc]: > times are based the timezone of the master

LGTM except seems should be `are based on`

**Comment by mburke5678** [comment]
Small change. No peer review needed.

**Pull request merged**